openverse_catalog/dags/provider_api_scripts/stocksnap.py

```diff
@@ -176,7 +176,7 @@ def _get_metadata(item):
     """
     Include popularity statistics.
     """
-    extras = ["downloads", "page_views", "favorites"]
+    extras = ["downloads_raw", "page_views_raw", "favorites_raw"]
```

Comment:
"downloads", "page_views", "favorites" automatically formatted numbers > 1000 as 1k, 2k and so on...these new `_raw` fields don't do that.